demisto_sdk/commands/unify_packs/packs_unifier.py

```diff
@@ -0,0 +1,191 @@
+import logging
+import os
+import sys
+from contextlib import contextmanager
+from pathlib import Path
+
+import click
+from demisto_sdk.commands.common.constants import PACKS_DIR
+from demisto_sdk.commands.common.content.objects.pack_objects.pack import Pack
+from demisto_sdk.commands.common.tools import arg_to_list
+from demisto_sdk.commands.create_artifacts.content_artifacts_creator import (
+    IGNORED_PACKS, ArtifactsManager, ContentObject, create_dirs, delete_dirs,
+    dump_pack, zip_pack_zips, zip_packs)
+
+EX_SUCCESS = 0
+EX_FAIL = 1
+
+
+class PacksUnifier:
+
+    def __init__(self, pack_paths: str, output: str, content_version: str, zip_all: bool, quite_mode: bool = False):
+        default_args = dict(packs=True, zip=True, cpus=os.cpu_count(), suffix='')
```

Comment:
Why did you choose `cpus=os.cpu_count()`?

Reply:
is a mandatory argument in ArtifactsManager - i changed it to pass cpus=1 as we don't use multi processing